scripts/linters/pylint_extensions.py

```diff
@@ -2919,6 +2919,48 @@ def visit_importfrom(self, node: astroid.nodes.ImportFrom) -> None:
                 self.add_message('disallowed-text-import', node=node)
 
 
+# TODO(#16567): Here we use MyPy ignore because of the incomplete typing of
+# pylint library and absences of stubs in pylint, forces MyPy to
+# assume that BaseChecker class has attributes of type Any.
+# Thus to avoid MyPy's error
+# (Class cannot subclass 'BaseChecker' (has type 'Any')),
+# we added an ignore here.
+class BlankLineBelowFunctionDefChecker(checkers.BaseChecker):  # type: ignore[misc]
+    """Checks if there a single blank line after function definition."""
```

Comment:
```suggestion
    """Ensures that there is no blank line after function definition."""
```

Let's make clear what the desired style is